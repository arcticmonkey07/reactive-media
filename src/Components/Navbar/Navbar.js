import React from 'react';
import { NavLink } from "react-router-dom";

const Navbar = () => {
  return (
    <nav className='navbar navbar-expand-lg navbar-dark bg-dark'>
      <div className='container-fluid'>
        <a className='navbar-brand' href='#'>
          Navbar
        </a>
        <button
          className='navbar-toggler'
          type='button'
          data-bs-toggle='collapse'
          data-bs-target='#navbarNavDropdown'
          aria-controls='navbarNavDropdown'
          aria-expanded='false'
          aria-label='Toggle navigation'
        >
          <span className='navbar-toggler-icon'></span>
        </button>
        <div className='collapse navbar-collapse' id='navbarNavDropdown'>
          <ul className='navbar-nav'>
            <li className='nav-item'>
              <NavLink className='nav-link active' aria-current='page' to='/'>
                Home
              </NavLink>
            </li>
            <li className='nav-item'>
              <NavLink className='nav-link' to='/create'>
                Create User
              </NavLink>
            </li>
          </ul>
        </div>
      </div>
    </nav>
  );
}

export default Navbar;
