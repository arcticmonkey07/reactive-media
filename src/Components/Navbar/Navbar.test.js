import renderer from 'react-test-renderer';
import Navbar from "./Navbar";

it("render correctly Navbar component", () => {
  const NavbarComponent = renderer.create(<Navbar />).toJSON();
  expect(NavbarComponent).toMatchSnapshot();
});