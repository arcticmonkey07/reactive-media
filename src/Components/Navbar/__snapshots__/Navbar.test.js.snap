// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Navbar component 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-dark bg-dark"
>
  <div
    className="container-fluid"
  >
    <a
      className="navbar-brand"
      href="#"
    >
      Navbar
    </a>
    <button
      aria-controls="navbarNavDropdown"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-bs-target="#navbarNavDropdown"
      data-bs-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarNavDropdown"
    >
      <ul
        className="navbar-nav"
      >
        <li
          className="nav-item"
        >
          <a
            aria-current="page"
            className="nav-link active"
            to="/"
          >
            Home
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            to="/create"
          >
            Create User
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
